packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -523,22 +523,24 @@ class Checkout extends Component<
     }
 
     private renderCartSummary(): ReactNode {
+        const { isMultiShippingMode } = this.state;
```

Comment:
In addtion to the manual switch, we also read consignments to determine if the checkout is using multi-shipping during each page initialization.
https://github.com/bigcommerce/checkout-js/blob/e49e97ca3cb6b0a5ee90f782ae2a0bee3e52dbcc/packages/core/src/app/checkout/Checkout.tsx#L259